index.md

```diff
@@ -1,11 +1,244 @@
+An example of this was which HiPS product to use for the coverage map, which field to center the coverage map to, etc.
+
+Additionally, it was unclear who could be negotiated with for discussions along the lines of "do you want to go live like this, or do you want us to slip and fix this first".
+The RSP product owners provided great input and context, however they did not have the authority to slip a community-promised date either.
+Lines of command are further blurred by the fact that a data release crosses the boundaries of Data Management and System Performance.
+
+**Recommendation:** A "decider of last resort" with the expertise and organisational standing to make quick authoritative decisions on appeal by the Data Services Lead should be designated.
+
+Note that while in operations there is a plan for a Data Release board, it is still necessary to have a person (perhaps the same person would would chair this board) that can make timely decisions without a formal process.
+
+### Insufficient internal red-team testing
+
+One of the reasons that it was unclear how close to the deadline we were going to cut it until it was almost too late was the very late involvement of "critical" testers.
+Organisationally there are two groups that can provide pre-release testing, however the V&V team focuses on pre-handover pipeline product testing, and the CST team focus on "best practice" testing for their educational materials.
+
+The majority of late (2-3 week pre-release) serious issues was uncovered by testers with high levels of data and software domain expertise -- Product Owners, Pipeline Developers, Middleware Developers.
+These had a priori knowledge of what the answer _should_ be, so they were excellent
+Many of these were testing to be helpful off their own bat (and we appreciate them).
+However, this activity requires more effort that can be applied on a volunteer basis or an ad-hoc timeline.
+
+**Recommendation:** An organized red-team testing campaign should form part of the release schedule with testers with strong a priori knowledge of desired system behavior (and even some knowledge of where the bodies are buried).
+
+Even one day of testing under such conditions, with a process for ticketing feedback, would be immensely valuable.
+
+### Absence of a cumulative corpus for automated testing
+
+From a technical point of view, it's oddly reassuring that our systems are mature enough for unscripted hand-offs.
+The downside of this is that the lack of process means that it's harder to set up automated testing against delivered data products.
+An example of this was the now infamous "where are the HiPS files" moment where it transpired it was unclear where the HiPS files were, where they should go, what the HiPS file delivery consisted of, whether it was a complete set etc.
+At moments there was deja-vu with issues seen during DP0.2.
```

Comment:
I think this undersells the importance of this point: we didn't apply the lessons of DP0.2 adequately, again mainly regarding the handoffs along the chain.

The warning lights were flashing before; they're flashing again now; we will not do well with DP2 -- or with the PP data release where there's virtually no re-work time at all -- if we don't apply them now.